nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -323,8 +323,13 @@ TfOpNodes buildOpNodes(const GraphRepresentation &representation,
 
     Exporter e(&fbb);
     ln->exportTo(e, ml::train::ExportMethods::METHOD_TFLITE);
+    auto export_output = e.getResult<ml::train::ExportMethods::METHOD_TFLITE>();
+
+    if (export_output.get()->getWeights().size() == 0) {
+      export_output.get()->setTrainable(false);
+    }
 
-    nodes.emplace_back(e.getResult<ml::train::ExportMethods::METHOD_TFLITE>());
+    nodes.emplace_back(move(export_output));
```

Comment:
I wonder do we need this move here? AFAIK, getResult return moved object already.

Reply:
I think yes.
getResult move object! but for some check i already move object to export_output, so we need to move again to nodes